### PR TITLE
[psen scan v2] Update parameters

### PR DESCRIPTION
**Pull request by martiniil**
## Description

Follow-up of pilzde/psen_scan_v2#196. Based on #26.

### CODE REVIEW

File: psen_scan_v2_tutorials/config/scanner2_config.yaml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # %Tag(FULLTEXT)%
-sensor_ip: "192.168.0.20"
-host_ip: "192.168.0.50"
-host_udp_port_control: 50505 
-host_udp_port_data: 50506 
-prefix: laser_2
+sensor_ip: "192.168.0.20"    # IP address of the laser scanner
+prefix: laser_2              # This name is prefixed to the TF, where the scanner is mounted
+host_udp_port_control: 50505 # Make sure that not the same ports are used as in the first scanner config.
```
Comment (jschleicher) [inline psen_scan_v2_tutorials/config/scanner2_config.yaml:4]:
```suggestion
host_udp_port_control: 50505 # Make sure to use different ports than in the first scanner config.
```

File: psen_scan_v2_tutorials/config/scanner2_config.yaml [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+host_udp_port_data: 50506
+## The following parameters are optional
+angle_start: -2.399          # The angle of the first measurement (-2.399 rad ~= -137.5 deg)
+angle_end: 2.399             # The angle of the last measurement (2.399 rad ~= 137.5 deg)
+intensities: false           # Receive intensity values
+resolution: 0.0017           # Scan resolution (0.0017 rad ~= 0.1 deg)
+host_ip: "192.168.0.50"      # Change the IP to the IP of your PC.
```
Comment (jschleicher) [inline psen_scan_v2_tutorials/config/scanner2_config.yaml:11]:
```suggestion
host_ip: "192.168.0.50"      # IP of your PC, if auto-detection doesn't work for you.
```

File: psen_scan_v2_tutorials/config/scanner2_config.yaml [hunk 3]
```diff
@@ -1,7 +1,13 @@
...
-prefix: laser_2
+sensor_ip: "192.168.0.20"    # IP address of the laser scanner
+prefix: laser_2              # This name is prefixed to the TF, where the scanner is mounted
+host_udp_port_control: 50505 # Make sure that not the same ports are used as in the first scanner config.
+host_udp_port_data: 50506
+## The following parameters are optional
+angle_start: -2.399          # The angle of the first measurement (-2.399 rad ~= -137.5 deg)
+angle_end: 2.399             # The angle of the last measurement (2.399 rad ~= 137.5 deg)
+intensities: false           # Receive intensity values
+resolution: 0.0017           # Scan resolution (0.0017 rad ~= 0.1 deg)
```
Comment (jschleicher) [inline psen_scan_v2_tutorials/config/scanner2_config.yaml:10]:
```suggestion
resolution: deg(0.1)         # Scan resolution (0.0017 rad ~= 0.1 deg)
```
Do python expressions work as described in http://wiki.ros.org/rosparam ?

File: psen_scan_v2_tutorials/config/scanner2_config.yaml [hunk 4]
```diff
@@ -1,7 +1,13 @@
...
-host_udp_port_control: 50505 
-host_udp_port_data: 50506 
-prefix: laser_2
+sensor_ip: "192.168.0.20"    # IP address of the laser scanner
+prefix: laser_2              # This name is prefixed to the TF, where the scanner is mounted
+host_udp_port_control: 50505 # Make sure that not the same ports are used as in the first scanner config.
+host_udp_port_data: 50506
+## The following parameters are optional
+angle_start: -2.399          # The angle of the first measurement (-2.399 rad ~= -137.5 deg)
+angle_end: 2.399             # The angle of the last measurement (2.399 rad ~= 137.5 deg)
```
Comment (jschleicher) [inline psen_scan_v2_tutorials/config/scanner2_config.yaml:8]:
```suggestion
angle_start: deg(-137.5)     # The angle of the first measurement (-2.399 rad ~= -137.5 deg)
angle_end: deg(137.5)        # The angle of the last measurement (2.399 rad ~= 137.5 deg)
```
see below

**Review by jschleicher — APPROVED** [review]
Thank you for bringing this up again! It's still hard to understand, that the ordering is different between the two configs. But let's postpone the port(s) discussion for future improvements.

See tiny wording suggestions inline.

File: psen_scan_v2_tutorials/config/scanner2_config.yaml (hunk 3 above)
Comment (martiniil) [reply psen_scan_v2_tutorials/config/scanner2_config.yaml:10]: If this works, update pilzde/psen_scan_v2/README.md, too

Comment (martiniil) [reply psen_scan_v2_tutorials/config/scanner2_config.yaml]: Yes, but these are special rosparam converters, not python ...

**Pull request merged**